### PR TITLE
Try to explain colClasses a bit more clearly

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
Thoughts on doing it this way?

### CODE REVIEW

File: content/intro-curriculum/B_Get.Rmd [hunk 1]
```diff
@@ -386,7 +386,15 @@ str(read.csv("data/course_NWISdata.csv"))
...
+Another issue with reading in USGS data is that site numbers often have a leading zero that is dropped if it defaults to an integer. To prevent this, you can specify the class for each column using the `colClasses` argument. In our dataset, we know that site numbers (column one) should be treated as character. We will define the other columns to match their data type. See `?read.table` for more information about `colClasses`.
+
+```{r, warning=FALSE, message=FALSE}
+column_types <- c("character", "character", "numeric", "character", "numeric", "numeric", "numeric")
+intro_df <- read.csv("data/course_NWISdata.csv", stringsAsFactors = FALSE, colClasses = column_types)
+str(intro_df)
+```
+
+As a shortcut, we can use `NA` for columns that we would like R to determine. If we aren't positive about the other six columns or are only concerned about enforcing the first, we can put `NA` in the arguments for `colClasses` to indicate we want `read.table` to choose for us.
```
Comment (wdwatkins) [inline content/intro-curriculum/B_Get.Rmd]: 👍

**Comment by jiwalker-usgs** [comment]
@lindsaycarr does this make it more clear?

**Comment by lindsaycarr** [comment]
Fixing #151

**Comment by wdwatkins** [comment]
@lindsaycarr @jiwalker-usgs 
I just realized you could do this via `fread`, but you can use a named vector to only change particular column types.  i.e. 
```
tread <- read.csv('cache/fetch/gaData.csv', colClasses = c(viewID = "character"))
```

It would avoid the whole same length vector thing, although then you have to introduce named vectors instead, so maybe it is a wash.

**Comment by lindsaycarr** [comment]
We'd have to introduce named vectors and fread (not that fread is complicated), but I'd like to stick to read.csv for beginners personally.

**Comment by wdwatkins** [comment]
It works with `read.csv` too - see ☝️.  I just discovered it in the course of using `fread`.

**Comment by lindsaycarr** [comment]
Oh whoops. Guess I didn't read that close enough! I'm wondering if that would be more confusing than typing out the whole vector to newbies. This seems potentially harder to conceptualize

**Comment by wdwatkins** [comment]
Yeah.  Just an option.

**Comment by ldecicco-USGS** [comment]
I don't think it's more confusing than having to load the whole thing twice just to find out how many columns there are. ESPECIALLY for USGS'ers...they need to do this any time there is a site id or parameter code.

**Pull request merged**